charts/seed-controlplane/charts/etcd/values.yaml

```diff
@@ -3,6 +3,11 @@ replicas: 1
 
 images:
   etcd: image-repository:image-tag
+  etcd-backup-restore: image-repository:image-tag
+
+
+etcdVersion: v3.3.17
```

Comment:
These two version keys are no longer needed as the image is injected above, correct? Please remove.

Reply:
Done